merlin/models/tf/__init__.py

```diff
@@ -108,8 +104,13 @@
 from merlin.models.tf.prediction_tasks.next_item import NextItemPredictionTask
 from merlin.models.tf.prediction_tasks.regression import RegressionTask
 from merlin.models.tf.prediction_tasks.retrieval import ItemRetrievalTask
+
+# Must happen before any importing of tensorflow to curtail mem usage
 from merlin.models.tf.utils import repr_utils
 
+configure_tensorflow()
```

Comment:
Why did this move? Isn’t this after the tensorflow imports now?

Reply:
ugh isort

Reply:
Something about the merge commit is...weird

Reply:
Judging by the stray comment down at the bottom I had to remove, seems like I'm not the first person this has happened to. 😅